config/grunt/task-config/aliases.yaml

```diff
@@ -0,0 +1,16 @@
+# Grunt aliases
+---
+'build':
+  - 'build:js'
+
+# Build JavaScript from assets to production
+'build:js':
+  - 'clean:build-assets'
+  - 'webpack:buildDev'
```

Comment:
I'm definitely not a webpack/grunt expert but I see in the webpack.js file there are two more tasks: watch and buildProd, but no grunt aliases for them. Is this because this setup is a first implementation for now and will be refined later?

Reply:
Clarified with Jimmy this isn't a problem for now.